docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to LoopBack
+
+Contributions to Node.js include code, documentation, answering user questions, and advocating for all types of Node.js users. See our official documentation on loopback.io for more information common to all of our GitHub repositories:
+
+  - http://loopback.io/doc/en/contrib/index.html
+
+## [Principles](http://loopback.io/doc/en/contrib/Governance.html#principles)
+
+LoopBack is an open, inclusive, and tolerant community of people working together to build a world-class Node framework and tools. We value diversity of individuals and opinions, and seek to operate on consensus whenever possible. We strive to maintain a welcoming, inclusive, and harassment-free environment, regardless of the form of communication. When consensus is not achievable, we defer to the owners of each individual module; the powers of the individual owner are kept in check by the ability of the community to fork and replace dependencies on the individual module and maintainer.
+
+## [Reporting issues](http://loopback.io/doc/en/contrib/Reporting-issues.html)
+
+Issues in [strongloop/loopback-next](https://github.com/strongloop/loopback-next/issues) are the primary means by which bug reports and general discussions are made.
+
+ - [How to report an issue](http://loopback.io/doc/en/contrib/Reporting-issues.html#how-to-report-an-issue)
+ - [Disclosing security vulnerabilities](http://loopback.io/doc/en/contrib/Reporting-issues.html#security-issues)
+
+## [Contributing code](http://loopback.io/doc/en/contrib/code-contrib.html)
+
+Pull Requests are the way concrete changes are made to the code, documentation
+and tools contained in LoopBack repositories.
+
+ - [Setting up development environment](./DEVELOPING.md#setting-up-development-environment)
+ - [How to contribute the code](http://loopback.io/doc/en/contrib/code-contrib.html#how-to-contribute-to-the-code)
+ - [Building the project](./DEVELOPING.md#building-the-project)
+ - [Running tests](./DEVELOPING.md#running-tests)
+ - [Coding rules](./DEVELOPING.md#coding-rules)
+ - [API documentation](./DEVELOPING.md#api-documentation)
+ - [Git commit messages](./DEVELOPING.md#commit-message-guidelines)
+ - [Reviewing pull requests](http://loopback.io/doc/en/contrib/triaging-pull-requests.html)
+ - [Contributor License Agreement (CLA)](http://loopback.io/doc/en/contrib/code-contrib.html#agreeing-to-the-cla)
+
+## [Documentation](http://loopback.io/doc/en/contrib/doc-contrib.html)
+
+LoopBack documentation is sourced in the [strongloop/loopback.io](https://github.com/strongloop/loopback.io) GitHub repository.
+
+ - [How LoopBack documentation works](http://loopback.io/doc/en/contrib/doc-contrib.html#how-loopback-documentation-works)
+ - [Using Jekyll](http://loopback.io/doc/en/contrib/jekyll_getting_started.html)
+ - [Authoring pages](http://loopback.io/doc/en/contrib/pages.html)
+ - [Translations](http://loopback.io/doc/en/contrib/translation.html)
+
```

Comment:
I'm thinking we might need to add the commit message guidelines to here as well.  
e.g. https://loopback.io/doc/en/contrib/git-commit-messages.html
There were quite a number of doc PRs at the end of last year that failed in the commit linter.

Reply:
LoopBack Next uses different rules for commit messages and @bajtos has included details for that in `DEVELOPING.md`.

Reply:
If the doc PR is in loopback.io, I think the commit message guideline from https://loopback.io/doc/en/contrib/git-commit-messages.html is still valid.  
But yes, if it's from one of the READMEs, then it will follow the one in loopback-next.

Reply:
I am going to add a link to that other page too.

```md
[Git commit messages](./DEVELOPING.md#commit-message-guidelines) (Note that other repositories like [strongloop/loopback.io](https://github.com/strongloop/loopback.io) use a different [convention](https://loopback.io/doc/en/contrib/git-commit-messages.html).)
```

Reply:
On the second thought, I am going to revert the change mentioned in my previous comment and move the link to loopback.io commit message guidelines to the actual text in DEVELOPING.md file. A new commit will come shortly.